FirebaseRemoteConfig/Sources/RCNConfigContent.m

```diff
@@ -128,7 +127,6 @@ - (void)loadConfigFromMainTable {
                   NSDictionary *activePersonalization, NSDictionary *defaultConfig) {
     self->_fetchedPersonalization = [fetchedPersonalization copy];
     self->_activePersonalization = [activePersonalization copy];
```

Comment:
Since we only have one dispatch_semaphore_signal blocking inside `[DBManager loadMainWithBundleIdentifier:]` to block the read, should we move this `[DBManager loadPersonalizationWithCompletionHandler:]` inside `[DBManager loadMainWithBundleIdentifier:]` as well?

Reply:
Feels a bit weird to chain calls to `DBManager` and add extra latency, especially since the personalization data is separate from the RC template and isn't returned as part of the config.

I think we could refactor `[DBManager loadMainWithBundleIdentifier:]` to return personalization data along with the config (in one call), but that's a larger change and we can tackle it as a follow up. The current change will fix the reported issue, and doesn't change current personalization behavior (we already aren't blocking)

Reply:
my concern is if developers try to read fetchedPersonalization then it might not be ready. I'm not familiar with personalization for but if we are okay that they might not be fully loaded when developers trying to read configs, i'm okay give it good to go.

Reply:
Can you add a todo here on refactor?

Reply:
Developers don't read the `fetchedPersonalization`s so it should be okay if they're not loaded when developers try to read configs. They're used to report personalization signals to GA when a personalized parameter (stored in the config that's read above and waited on) is read. More signals improve the personalizations but some missed signals don't break the system.

Reply:
added a TODO for the personalization refactor